index.html

```diff
@@ -171,15 +170,59 @@ <h2>Multi-Post Stories </h2>
                 <button class="button" type="button" onclick="">See Project</button>
             </div>
         </div>
-
-
-
-
-
-
-
-
-
+<footer>
+    <div class="template-footer">
+        <h1>CONTACT US</h1>
+        <p>If you have an application you are <br>
```

Comment:
- [ ] It seems that your 'Inter' font is not working in this piece of text. Double-check if you imported the font correctly. This section looks wider on the Figma template. Please apply those changes to make your app looks like the template.
 -Your design:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/61919617/186493034-6becd443-13d7-49a2-8a2e-3edf23d0d647.png">
-Figma template:
<img width="396" alt="image" src="https://user-images.githubusercontent.com/61919617/186493107-7baee542-e46c-453f-8d2c-b005f57f6ceb.png">